Tasks/SonarQubePostTest/SonarQubePostTest.ps1

```diff
@@ -34,4 +34,23 @@ else
      Write-Warning "Could not find the summary report file $summaryMdPath"
 }
 
+. .\CodeAnalysisFilePathComputation.ps1
+$prcaEnabled = Get-TaskVariable -Context $distributedTaskContext -Name "GitPullRequestCodeAnalysisEnabled"
```

Comment:
Would it make sense to check the result of *Invoke-BatchScript* before attempting to post-process the json file? If the MSBuild Runner process failed then it might be better to early-out rather than continuing (which will fill up the log with additional messages that will make diagnosing the real failure harder).